recipes/mcap/all/conanfile.py

```diff
@@ -10,7 +13,7 @@ class McapConan(ConanFile):
     url = "https://github.com/conan-io/conan-center-index"
     homepage = "https://github.com/foxglove/mcap"
     description = "A C++ implementation of the MCAP file format"
-    license = "Apache-2.0"
+    license = "MIT"
```

Comment:
Was it wrong or was the license updated?

Reply:
So older version are still under Apache-2.0 and only since this new release is using MIT. You should configure it properly:

```python
def configure(self):
    if Version(self.version) < "0.3.0":
        self.license = "Apache-2.0"
```